docs/tutorial/efficient-dataflow.md

```diff
@@ -157,19 +168,19 @@ Then we add necessary transformations:
 .. code-block:: python
     :emphasize-lines: 3-5
 
-    ds = LMDBData(db, shuffle=False)
+    ds = LMDBData(db, shuffle=False) # produces key-value pairs
     ds = LocallyShuffleData(ds, 50000)
-    ds = LMDBDataPoint(ds)
-    ds = MapDataComponent(ds, lambda x: cv2.imdecode(x, cv2.IMREAD_COLOR), 0)
+    ds = LMDBDataPoint(ds)  # converts 'ds' to  pairs (encoded_image, label)
+    ds = ImageDecode(ds, mode='.jpg', index=0)
```

Comment:
Can we just keep the Map? It's a one-liner anyway, and user can see and choose what decoder they want (I know faster decoder than opencv). The wrappers also lose the options available in imdecode/imencode (IMREAD mode, image quality, etc).